### PR TITLE
Add PRQL

### PR DESCRIPTION
**Pull request by vanillajonathan**
PRQL is a modern language for transforming data — a simple, powerful, pipelined SQL replacement.

https://prql-lang.org/
https://github.com/PRQL/prql

### CODE REVIEW

**Review by JoaoDanielRufino — CHANGES_REQUESTED** [review]
Hi, thanks for the initiative!

Could you fix the tests ?

**Comment by vanillajonathan** [comment]
I don't know how to. The diff looks clean and the logs are not very telling. Do you know what is wrong?

File: README.md [hunk 1]
```diff
@@ -192,6 +192,7 @@ $ gcloc languages
   Processing       |  .pde          |  //             |  /* */
   Protocol Buffers |  .proto        |  //             |
   Python           |  .py           |  #              |  """ """
+  PRQL             |  .prql         |  #              |
```
Comment (JoaoDanielRufino) [inline README.md:195]: Could you add `PRQL` between `Protocol Buffers` and `Python` please ?

**Review by JoaoDanielRufino — CHANGES_REQUESTED** [review]

**Comment by JoaoDanielRufino** [comment]
> I don't know how to. The diff looks clean and the logs are not very telling. Do you know what is wrong?

Take a look at files changes of [this PR](https://github.com/JoaoDanielRufino/gcloc/pull/66)

You also need to add the expected results on the other tests of `analyzer_test.go` and `scanner_test.go`

File: README.md (hunk 1 above)
Comment (vanillajonathan) [reply README.md]: Done!

File: README.md [hunk 2]
```diff
@@ -190,6 +190,7 @@ $ gcloc languages
   Plain Text       |  .txt .text    |                 |
   PowerShell       |  .ps1          |  #              |
   Processing       |  .pde          |  //             |  /* */
+  PRQL             |  .prql         |  #              |
   Protocol Buffers |  .proto        |  //             |
   Python           |  .py           |  #              |  """ """
```
Comment (JoaoDanielRufino) [inline README.md:195]: You added between `Processing` and `Protocol Buffers`, could you add `PRQL` between `Protocol Buffers` and  `Python` ?
```suggestion
  Processing       |  .pde          |  //             |  /* */
  Protocol Buffers |  .proto        |  //             |
  PRQL             |  .prql         |  #              |
  Python           |  .py           |  #              |  """ """
```

**Review by JoaoDanielRufino — CHANGES_REQUESTED** [review]

**Review by JoaoDanielRufino — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**